source/_docs/drupal-security-patches.md

```diff
@@ -53,6 +53,7 @@ You'll need a local git clone of your site code.
 
     ```bash
     rm 206.patch
+    git add core/lib/Drupal/Core/Security/
```

Comment:
There are files in other locations that also need to be added. Can we just say `git add .` instead?